tests/foreman/cli/test_satellitesync.py

```diff
@@ -271,28 +431,32 @@ def _create_cv(cv_name, repo, organization, publish=True):
     return content_view, cvv_id
 
 
-def _enable_rhel_content(organization, repo_name, releasever=None, product=None, sync=True):
+def _enable_rhel_content(
+    default_sat, module_org, repo_name, releasever=None, product=None, sync=True
```

Comment:
same comment about using fixture names as argument names for non-fixture functions :)